src/main/scala/org/scalajs/tools/tsimporter/Config.scala

```diff
@@ -0,0 +1,27 @@
+package org.scalajs.tools.tsimporter
+
+
+case class Config(
+  inputFileName: String = "",
+  outputFileName: String = "",
+  packageName: String = "importedjs"
```

Comment:
Those lines should be indented 4 spaces, not 2. (see https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#indentation).